exercises/isbn-verifier/canonical-data.json

```diff
@@ -0,0 +1,82 @@
+{
+    "exercise": "isbn-verifier",
+    "version": "1.0.0",
+    "comments": [
+        "An expected value of true indicates a valid ISBN-10, ",
+        "whereas false means the ISBN-10 is invalid."
+    ],
+    "cases": [
+        {
+            "description": "valid isbn number",
+            "property": "isbn",
+            "input": "3-598-21508-8",
+            "expected": true
+        },
+        {
+            "description": "invalid isbn check digit",
+            "property": "isbn",
+            "input": "3-598-21508-9",
+            "expected": false
+        },
+        {
+            "description": "valid isbn number with a check digit of 10",
+            "property": "isbn",
+            "input": "3-598-21507-X",
+            "expected": true
+        },
+        {
+            "description": "check digit is a character other than X",
+            "property": "isbn",
+            "input": "3-598-21507-A",
+            "expected": false
+        },
+        {
+            "description": "invalid character in isbn",
+            "property": "isbn",
+            "input": "3-598-2K507-0",
+            "expected": false
+        },
+        {
+            "description": "valid character in isbn",
+            "property": "isbn",
+            "input": "3-598-2X507-0",
+            "expected": false
+        },
+        {
+            "description": "valid isbn without separating dashes",
+            "property": "isbn",
+            "input": "3598215088",
+            "expected": true
+        },
+        {
+            "description": "isbn without separating dashes",
```

Comment:
This description is the same as the last, what is this test testing differently?

Reply:
The X is the checkdigit